prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -97,13 +97,12 @@ object FhirPathUtils : Logging {
             } else {
                 pathEngine.evaluate(appContext, focusResource, bundle, bundle, expressionNode)
             }
-        } catch (e: Exception) {
-            // This is due to a bug in at least the extension() function
-            logger.error(
-                "Unknown error while evaluating FHIR expression $expression. " +
-                    "Returning empty resource list.",
-                e
-            )
+        } catch (e: FHIRLexerException) {
```

Comment:
did you look into if we could get these exceptions to print pretty? It is a bit out of scope and not all that urgent since it's an internal tool. But would be nice

Reply:
Yeah good point, especially using the cli fhirpath tool it's ugly. Fixed it by just using the java class name, let me know if that is alright.

Reply:
Looks better to me!